### PR TITLE
Add a new full width layout

### PR DESCRIPTION
**Pull request by sherbastian**
## Description

- Added a new Plain layout that does not include navigation menus.
- Altered the way that the layout is selected on page load, the layout selection is determined based on route 'layout' attribute and in absence will default to the Platform layout.
- Altered the log polling functionality which was based on route paths to again use a meta attribute 'shouldPoll' adding the possibility to fine tune the routes that use it.
- Extracts `Alerts` logic into a standalone mixin

## Related Issue(s)

required in #3656 

## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] Suitable unit/system level tests have been added and they pass 
 - [ ] Documentation has been updated
    - [ ] Upgrade instructions
    - [ ] Configuration details
    - [ ] Concepts
 - [ ] Changes `flowforge.yml`?
    - [ ] Issue/PR raised on `FlowFuse/helm` to update ConfigMap Template
    - [ ] Issue/PR raised on `FlowFuse/CloudProject` to update values for Staging/Production

## Labels

 - [ ] Backport needed? -> add the `backport` label
 - [ ] Includes a DB migration? -> add the `area:migration` label

### CODE REVIEW

**Comment by sherbastian** [comment]
The existing layout functionality should not be impacted as no pages currently use the plain layout. This being said, the setupApp should also be tested.
In order to check the new plain layout, one could add the 'layout: "plain"' meta attribute to a page (one that preferably does not have teleport components) and check that the page loads without any menus. Additionally, alerts and modals should be tested that they work on the new layout.

File: frontend/src/pages/instance/components/InstanceLogs.vue [hunk 1]
```diff
@@ -101,7 +100,9 @@ export default {
     },
     methods: {
         shouldPoll: function () {
-            return this.$route.fullPath.match(/\/(?:application|instance)\/[^/]+\/logs/)
+            return Object.hasOwnProperty.call(this.$route, 'meta') &&
```
Comment (joepavitt) [inline frontend/src/pages/instance/components/InstanceLogs.vue:103]: why not `this.$route.hadsOwnProperty('meta')`?

Comment (sherbastian) [reply frontend/src/pages/instance/components/InstanceLogs.vue:103]: Force of habit mainly, but also protects against this.$route being undefined or null

File: frontend/src/pages/instance/components/InstanceLogs.vue [hunk 2]
```diff
@@ -205,15 +206,3 @@ export default {
     }
 }
 </script>
-
-<style scoped>
-.forge-log-offline-background {
```
[the remark this answers is not in the document]
Comment (sherbastian) [reply frontend/src/pages/instance/components/InstanceLogs.vue:210]: My ide marked it as unused and I missed the fact that it was used dynamically. Reverted

**Review by joepavitt — APPROVED** [review]

**Pull request merged**